### ISSUE TITLE
JoinStep subsearch with hdfs load got exception 

### ISSUE THREAD

**Issue by elliVM** [teragrep/pth_10#160]
Tried to join a saved HDFS file to dataset with a column used in bloom operations.

 maybe off by one error

## Tried
```
index=search_bench earliest="11/27/2020:17:00:00"
| join partition [teragrep exec hdfs load /tmp/myFile] | teragrep exec bloom create
```
## Workaround that worked
```
index=search_bench earliest="11/27/2020:17:00:00"
| join partition [teragrep exec hdfs load /tmp/myFile | table partition, count] | teragrep exec bloom create
```
## Error from tried
```
java.lang.RuntimeException: SubsearchStep doesn't have any steps to execute!
	at com.teragrep.pth10.steps.subsearch.SubsearchStep.get(SubsearchStep.java:80)
	at com.teragrep.pth10.steps.join.JoinStep.get(JoinStep.java:77)
	at com.teragrep.pth10.ast.StepList.executeFromStep(StepList.java:180)
	at com.teragrep.pth10.ast.StepList.execute(StepList.java:150)
	at com.teragrep.pth10.ast.DPLParserCatalystVisitor.visitRoot(DPLParserCatalystVisitor.java:285)
	at com.teragrep.pth10.ast.DPLParserCatalystVisitor.visitRoot(DPLParserCatalystVisitor.java:80)
	at com.teragrep.pth_03.antlr.DPLParser$RootContext.accept(DPLParser.java:3070)
	at com.teragrep.pth_03.shaded.org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
	at com.teragrep.pth_07.DPLExecutor.interpret(DPLExecutor.java:185)
	at com.teragrep.pth_07.DPLInterpreter.internalInterpret(DPLInterpreter.java:159)
	at org.apache.zeppelin.interpreter.AbstractInterpreter.interpret(AbstractInterpreter.java:47)
	at org.apache.zeppelin.interpreter.LazyOpenInterpreter.interpret(LazyOpenInterpreter.java:110)
	at org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer$InterpretJob.jobRun(RemoteInterpreterServer.java:860)
	at org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer$InterpretJob.jobRun(RemoteInterpreterServer.java:752)
	at org.apache.zeppelin.scheduler.Job.run(Job.java:172)
	at org.apache.zeppelin.scheduler.AbstractScheduler.runJob(AbstractScheduler.java:132)
	at org.apache.zeppelin.scheduler.FIFOScheduler.lambda$runJobInScheduler$0(FIFOScheduler.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
```

**Comment by elliVM** [comment]
myFile had 2 columns "partition" and "estimate(tokens)"

**Comment by eemhu** [comment]
should be fixed with PR #560
